client/src/components/commonComponents/TabsCheckout/FormModel/checkoutFormModel.js

```diff
@@ -0,0 +1,25 @@
+export default {
+  formId: 'checkoutForm',
+  formField: {
+    fullName: {
+      name: 'fullName',
+      label: 'full Name',
```

Comment:
Pay attention to the capitalization in the labels here, the form isn't totally consistent, I'd change
'full Name' -> 'Full Name',
'email' -> 'Email',
zip/Postal Code -> Zip/Postal Code

Reply:
Fixed :+1: